File: DSA/Assignment_3/num_check.c
#include <stdio.h>
int main()
{
    int p=0,n=0,e=0,o=0;
    printf("Enter 25 numbers-\n");
    for(int i=0;i<25;i++)
    {
        printf("Enter a number:");
    }
    for(int i=0;i<25;i++)
    {
        if(i>0)
        p++;
        if(i<0)
        n++;
        if(i%2==0)
        e++;
        if(i%2!=0)
        o++;
    }
    printf("No. of positive numbers:%d\n",p);
    printf("No. of negative numbers:%d\n",n);
    printf("NO. of even numbers:%d\n",e);
    
    return 0;
}

File: DSA/Assignment_3/num_check.py
print("Enter 25 numbers:")
for i in range(25):
    n=int(input("Enter a number:"))
    if n>0:
        pos+=1
    if n<0:
        neg+=1
    if n%2==0:
        even+=1
    if n%2!=0:
        odd+=1

print(f"Positive:{pos}\nNegaitve:{neg}\nEven:{even}\nOdd:{odd}")